config/default.yml

```diff
@@ -2,183 +2,66 @@ require:
   - rubocop/cop/github
   - rubocop-performance
 
-AllCops:
-  DisabledByDefault: true
-
-Bundler/DuplicatedGem:
-  Enabled: true
-
-Bundler/OrderedGems:
-  Enabled: true
-
-GitHub/InsecureHashAlgorithm:
-  Enabled: true
+Gemspec/DependencyVersion:
+  Enabled: false
```

Comment:
For some of these new ones that fail _in the code in this gem_: should we put them in here, the global config for the gem and the one that all its downstream consumers use, or should we put them in `.rubocop.yml` inside this repo only?

I guess I'm now concerned, after @bensheldon's comments, about this gem's config not being a reflection of the styleguide? Who gets to make decisions on what we include in the styleguide, and hence who gets to "control" what sensible default RuboCop configuration we send out into the world?